openpype/hosts/houdini/plugins/publish/collect_cache_farm.py

```diff
@@ -0,0 +1,87 @@
+import os
+import pyblish.api
+import hou
+from openpype.hosts.houdini.api import lib, colorspace
+from openpype.hosts.houdini.api.lib import get_color_management_preferences
+
+
+class CollectDataforCache(pyblish.api.InstancePlugin):
+    """Collect data for caching to Deadline."""
+
+    order = pyblish.api.CollectorOrder + 0.04
+    families = ["ass", "pointcache",
+                "mantraifd", "redshiftproxy"]
+    hosts = ["houdini"]
+    targets = ["local", "remote"]
+    label = "Collect Data for Cache"
+    chunkSize = 999999
+
+    def process(self, instance):
+        if instance.data.get("farm", True):
+            for family in instance.data["families"]:
+                instance.data["family"] = family
+                if family == "bgeo" or "abc":
+                    instance.data["family"] = "pointcache"
+                instance.data["subset"] = instance.name
```

Comment:
This is a mess of broken code. Sorry ;)

```python
if family == "bgeo" or "abc"
```
equals:
```python
if family == True:
```
Not what you intended ;)
```python
if family in {"bgeo", "abc"}
```
I think is what you meant.

Also, you're setting subset to `instance.name` during the for loop which is weird to begin with - since in the for loop it's always the same value. But why do you need to override subset anyway - seems like a bug?

It feels like all you were trying to do is that _if_ bgeo or abc is the family that you want to label it pointcache. But why is that needed? What's the goal?

Reply:
I am struggling a little bit as both bgeo and abc share the same pointcache families.

Reply:
Struggling why? What issues are you facing with it?